Composer/packages/lib/shared/src/types/shell.ts

```diff
@@ -1,38 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { Diagnostic as LGDiagnostic } from 'botbuilder-lg';
+import { LGTemplate as LgTemplate } from 'botbuilder-lg';
+import { DialogInfo, LgFile, LuFile } from '@bfc/indexers';
 
-import { MicrosoftAdaptiveDialog } from './sdk';
-
-export interface FileInfo {
-  name: string;
-  content: string;
-  path: string;
-  relativePath: string;
-}
-
-export interface ITrigger {
-  id: string;
-  displayName: string;
-  type: string;
-  isIntent: boolean;
-}
-
-export interface DialogInfo {
-  content: MicrosoftAdaptiveDialog;
-  diagnostics: string[];
-  displayName: string;
-  id: string;
-  isRoot: boolean;
-  lgFile: string;
-  lgTemplates: string[];
-  luFile: string;
-  luIntents: string[];
-  referredDialogs: string[];
-  relativePath: string;
-  triggers: ITrigger[];
-}
+export * from '@bfc/indexers/lib/type';
```

Comment:
Is this here just reduce churn in this PR? Otherwise, why not have the consuming packages just import the types directly?